plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -184,6 +184,8 @@ class chain_plugin_impl {
    methods::get_block_by_id::method_type::handle                     get_block_by_id_provider;
    methods::get_head_block_id::method_type::handle                   get_head_block_id_provider;
    methods::get_last_irreversible_block_number::method_type::handle  get_last_irreversible_block_number_provider;
+   methods::get_controller::method_type::handle                      get_controller_provider;
+   methods::get_abi_serializer_max_time::method_type::handle         get_abi_serializer_max_time_provider;
```

Comment:
`abi_serializer_max_time` I think can be grabbed from command line for plugings. See how mongo_db_plugin accesses it. I don't think we need a `methods` for it.